lib/Settings/MappingProfiles/listTemplate.js

```diff
@@ -7,8 +7,12 @@ import { listTemplate as defaultListTemplate } from '../../ListTemplate';
 export const listTemplate = {
   ...defaultListTemplate,
   folioRecord: record => {
-    const { existingRecordType } = record;
+    const { recordTypes } = record;
 
-    return <FormattedMessage id={FOLIO_RECORD_TYPES[existingRecordType].captionId} />;
+    return recordTypes.map((recordType, index) => {
```

Comment:
Let's simplify a bit.
```suggestion
    return recordTypes
      .map((recordType, index) => <FormattedMessage id={FOLIO_RECORD_TYPES[recordType].captionId} />)
      .join(', ')
```

Reply:
I tried this issue, but it returns `[object Object]`

Reply:
Ah, yes, fair enough. Sounds good then, then make sure to cover with tests this logic. Ideally to end up with 100% percent. For unit test approach which we have there, it should not be a problem.